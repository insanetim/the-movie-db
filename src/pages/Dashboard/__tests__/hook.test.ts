import { renderHook } from '@testing-library/react'
import Wrapper from 'src/utils/testHelpers/wrapperMock'

import useContainer from '../hook'

describe('Dashboard useContainer hook', () => {
  it('should match snapshot', () => {
    const { result } = renderHook(() => useContainer(), { wrapper: Wrapper })

    expect(result.current).toMatchSnapshot()
  })
})
