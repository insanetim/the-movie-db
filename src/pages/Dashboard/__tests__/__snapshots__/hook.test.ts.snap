// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard useContainer hook should match snapshot 1`] = `
{
  "query": "",
}
`;
